src/exposurelog/routers/find_exposures.py

```diff
@@ -104,9 +140,18 @@ async def find_exposures(
         "observation_types",
     )
 
-    conditions = []
+    # Work around two daf_butler registry.queryDimensionRecords bugs:
+    # * Specifying the instrument argument results in a LookupError
+    #   if the instrument is unknown. Specifying the instrument in ``where``
+    #   correctly returns no records, instead.
+    # * ``instrument`` cannot be specified in the ``bind`` argument.
```

Comment:
Can you put the ticket number for the fix in the comment as a hint to come back to it later?

Reply:
OK. Though this code works so I wasn't planning to change it back.